src/main/webapp/app/stores/history-tab.store.tsx

```diff
@@ -0,0 +1,60 @@
+import { action, computed, makeObservable, observable } from 'mobx';
+import _ from 'lodash';
+
+export class HistoryTabStore {
```

Comment:
I don't think we gonna share the history tab store and it should be placed with the tab component together.